src/guards/client-workspace-permission.guard.ts

```diff
@@ -19,6 +21,10 @@ export class ClientWorkspacePermissionGuard implements CanActivate {
     const request = context.switchToHttp().getRequest()
     const { workspaceId, clientId } = request.params
 
+    if (!isUUID(clientId)) {
```

Comment:
Let's only use guards for core functionalities such as checking if:
- is user authenticated
- a user has a particular role (hasRole)
- a user can access a workspace
- a user can manage (is admin) a workspace